fullscreen.bs

```diff
@@ -295,18 +295,28 @@ attribute's getter must run these steps:
  <li><p>Return null.
 </ol>
 
+<p>A <a>document</a> is said to be a <dfn>simple fullscreen document</dfn> if its <a>top layer</a>
```

Comment:
That should still be a simple fullscreen document, since only one fullscreen element is in the top layer.